datasets/doc/source/contributor-how-to-contribute-dataset.rst

```diff
@@ -0,0 +1,57 @@
+How to contribute a dataset
+===========================
+
+To make a dataset available in Flower Dataset (`flwr-datasets`), you need to add the dataset to `HuggingFace Hub <https://huggingface.co/>`_ .
+
+This guide will explain the best practices we found when adding datasets ourselves and point to the HFs guides. To see the datasets added by Flower, visit https://huggingface.co/flwrlabs.
+
+Dataset contribution process
+----------------------------
+The contribution contains two steps:
+
+1. Create the dataset locally.
+
+We recommend that you do not upload custom scripts to HuggingFace Hub; instead, create the dataset locally and upload the data, which will speed up the processing time each time the data set is downloaded.
```

Comment:
useful info, how about having it as a `:: tip:` right under the `Creating a datasets locally` section?